### ISSUE TITLE
homebrew package does not retain version info

### ISSUE THREAD

**Issue by beckettsean** [influxdb/influxdb#2667]
```
sean:/usr/local/Cellar/influxdb/0.9.0-rc30/bin$ ./influxd version
[srvr] 2015/05/26 16:37:35 InfluxDB v0.9 (git: unknown)
```

Installed InfluxDB RC27 with Homebrew, later upgraded to RC30 using brew. In both cases the version string is "unknown" rather than the correct build.

**Comment by toddboom** [comment]
@gunnaraasen i think i figured this one out - change the first build step from `build` to `install` and remove the second step. i think it's nuking settings made in the first one. we can take a look at it tomorrow if that doesn't make sense.

**Comment by gunnaraasen** [comment]
@toddboom good catch. That seems to work. I also bumped up the version to rc31.

This should be fixed as soon as Homebrew/homebrew#40135 is merged.

**Comment by toddboom** [comment]
Awesome!
